progress.py

```diff
@@ -104,7 +104,7 @@ def GetNonMatchingSize(path):
 bootPct = 100 * boot / bootSize
 ovlPct = 100 * ovl / ovlSize
 
-bytesPerHeartPiece = total / 80
+bytesPerHeartPiece = int(total / 80)
```

Comment:
Python has `//` for integer divisions, which I think it is a bit cleaner
```suggestion
bytesPerHeartPiece = total // 80
```